src/pacmod3_core.cpp

```diff
@@ -288,8 +288,8 @@ void GlobalRptMsg::parse(const std::vector<uint8_t> & in)
 {
   enabled = in[0] & 0x01;
   override_active = ((in[0] & 0x02) > 0);
-  fault_active = ((in[0] & 0x80) > 0);
-  config_fault_active = ((in[1] & 0x01) > 0);
+  system_fault_active = ((in[0] & 0x80) > 0);
+  config_fault_active = ((in[1] & 0x80) > 0);
```

Comment:
Any reason to change it from `0x01` to `0x80`?

Reply:
This was actually a parsing bug I happened to notice while looking at the DBC for field name changes.
I didn't bother to review the rest of the parsing code because I intend to replace it all with auto-generated code in a future version.